src/_articles/archive/zones.md

```diff
@@ -193,16 +213,13 @@ they still execute.
 As a consequence a zoned error handler might
 be invoked multiple times.
 
-Also note that an _error zone_—a zone that has an error handler—might
-handle errors that originate in a child (or other descendant)
-of that zone.
+A zoned error handler exists in an _error zone_.
+An error zone might handle errors that originate
+in a descendant of that zone.
 A simple rule determines where
 errors are handled in a sequence of future transformations
 (using `then()` or `catchError()`):
-
-{{site.alert.note}}
-  Errors on Future chains never cross the boundaries of error zones.
-{{site.alert.end}}
+Errors on Future chains never cross the boundaries of error zones.
```

Comment:
**suggestion:** Consider framing this as to why the reader would need an error zone, then define an error zone.

Reply:
This part isn't about why you'd need error zones; "error zone" is just a zone that has an error handler, which is defined way earlier (it's basically the point of the whole page up to here). 

This is just to introduce the term "error zone" because it helps conceptualize the fact that a handler might end up handling errors from descendent zones, and how to assess the chain of handling.

Reply:
**reply:** Maybe consider defining this with a little more clarity.
```
An _error zone_ covers any zone with an error handler.
```

...that's... not great on my part. But do you see the point?